### ISSUE TITLE
UserAvatar: Design review

### ISSUE THREAD

**Issue by matthewgallo** [carbon-design-system/ibm-products#5543]
This issue is to be completed by the community contributor for User Avatar 
Marion Hekeler (@marion-bruells)
@cate-wilcox

### Standards

- [ ] All pattern updates/changes/iterations have been discussed with the component developer
- [ ] Patterns have been approved by either DSAG or another approving entity

### Pattern and behavior

- [ ] The component behaves according to the guidelines set by the component maintainer
- [ ] The component’s UI matches the pattern specifications set by the pattern maintainer
- [ ] The component’s motion matches the specifications set by the pattern maintainer(s)
- [ ] The UI produced is responsive, cross-browser, and responds to the currently set Carbon theme.
- [ ] Colors, themes, sizes and additional props are true and correct, aligning with Carbon set tokens (unless otherwise specified by the pattern)
- [ ] Paddings/margins/spacings are true and correct, in both desktop and mobile views

### Storybook

- [ ] A functioning component renders in Storybook
- [ ] The Storybook displays multiple scenarios that show how the component can be used
- [ ] Changing props in the Storybook updates the component

### Final review by product PAL maintainer
- [ ] Check off [this list](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/wiki/Definition-of-done-(Visual-QA-checklist))
- [ ] Let Product PAL maintainers know when you are ready for a review in this issue

**Comment by aubrey-oneal** [comment]
@matthewgallo Could we get a link to the PR? Thanks

**Comment by matthewgallo** [comment]
This review would be to review the component as it exists currently in [our storybook](https://ibm-products.carbondesignsystem.com/?path=/docs/ibm-products-components-user-avatar-useravatar--docs). That way we can move it from canary to released.

**Comment by elycheea** [comment]
@marion-bruells @cate-wilcox Would you be able to perform a design review on the `UserAvatar`? We usually would use a spec. I think the closest we have now are #4017 and #4016. If either of you has the original Sketch files, that would be a help too since some of the images in the website might need updating.

https://pages.github.ibm.com/carbon/ibm-products/components/user-avatar/usage/#accessibility
Noticed that these are using hexcodes rather than text tokens.

**Comment by marion-bruells** [comment]
@elycheea I did an update on this really long time ago, but I think the PR never got merged and lost. https://github.ibm.com/CDAI-design/pal/pull/1600 Overall it looks good to me with all the changes on the site and I also reviewed the coded component.

The only change I suggest on the site (like you mentioned): change hex codes from #00000 and #FFFFFF to $text-primary considering the light/dark theme. 

I am sure I have somewhere an old sketch file with the images. I’ll try to find it :)
